### PR TITLE
refine: vim.notify user when on unsupported filetype

### PR DESCRIPTION
**Pull request by younger-1**

### CODE REVIEW

**Comment by crispgm** [comment]
This looks fine. I wonder if we should use `vim.log.levels.ERROR` or `vim.log.levels.WARN`?

File: lua/telescope/_extensions/heading.lua [hunk 1]
```diff
@@ -34,7 +34,7 @@ local function get_headings()
     )
     local ok, mod = pcall(require, mod_path)
     if not ok then
-        error('The file type is not supported by telescope-heading')
+        vim.notify('The file type is not supported by telescope-heading', vim.log.levels.WARN, { title = 'Telescope heading' })
```
Comment (crispgm) [inline lua/telescope/_extensions/heading.lua]: This line is too long, as the lint said :)

**Comment by younger-1** [comment]
> This looks fine. I wonder if we should use `vim.log.levels.ERROR` or `vim.log.levels.WARN`?

Both is ok to me. 😄

**Comment by crispgm** [comment]
@younger-1 WARN looks warmer. Merged.

**Pull request merged**